developerportal/apps/topics/migrations/0055_topic_relevant_events.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.14 on 2020-07-03 13:05
```

Comment:
This files' changes are from another PR which was based off this one, which I should have held off merging, but didn't... Sorry.

Ignore for review